### PR TITLE
Unable to scroll and bottom padding #54

### LINKED ISSUE #54: Unable to scroll and bottom padding
This issue is related to unable to scroll and the bottom padding to last grid.

I am a GSSOC'21 participant please assign this issue to me i would like to in this issue. 

**Screenshots**

image

### PR DESCRIPTION
**Pull request by Alik-Kumar-Ghosh**
## Fixes Issue Unable to scroll and bottom padding #54

- This issue is about adding scroll function and bottom padding so that the last cell may be visible.

Closes: #54

### The changes you've made

1.index.html
        added a div for bottom padding.
2.style.css
         removed overflow hidden to make it scroll able.
 
### Screenshot

Before

image

After

image

### CODE REVIEW

**Comment by ankitapuri** [comment]
@rish-singhal please review it once

File: css/style.css [hunk 1]
```diff
@@ -170,4 +169,6 @@ body {
     background-color: rgba(255, 85, 0, 1);
 }
 
-   
\ No newline at end of file
+#b-margin{
```
Comment (rish-singhal) [inline css/style.css:172]: Everything seems fine, add an empty line at the end of the style.css for good practices.

File: index.html [hunk 2]
```diff
@@ -31,6 +31,7 @@ <h2> Choose Grid Size </h2>
       </form>
       <h2> Design Canvas </h2>
       <table id="pixel_canvas"> </table>
+      <div id="b-margin"></div>
       <script src="js/app.js"></script>
```
Comment (rish-singhal) [inline index.html]: same with this, add a single line at the end. Thanks for your contribution! @Alik-Kumar-Ghosh

File: css/style.css (hunk 1 above)
Comment (Alik-Kumar-Ghosh) [reply css/style.css:172]: Please check made the changes needed.

File: index.html (hunk 2 above)
Comment (Alik-Kumar-Ghosh) [reply index.html]: Please check made the changes needed.

Comment (rish-singhal) [reply index.html]: I meant at the **end of the file** i.e. after </html> -- an empty line, you can undo the changes that you did.

Comment (Alik-Kumar-Ghosh) [reply index.html]: ok

**Comment by ankitapuri** [comment]
Sorry there were some commits which I had to merge on urgent basis . Please resolve the conflicting files.

**Pull request merged**